packages/react/src/components/Copy/Copy.tsx

```diff
@@ -58,8 +108,16 @@ export default function Copy({
 
   const initialLabel = other['aria-label'] ?? '';
 
+  // TODO: remove these two lines when IconButton is properly typed, revert TypedIconButton component to IconButton
+  type IconButtonType = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+    closeOnActivation?: boolean;
+    align: string;
+    label: string | undefined;
+  };
+  const TypedIconButton = IconButton as React.FC<IconButtonType>;
+
   return (
-    <IconButton
+    <TypedIconButton
```

Comment:
IconButton is not currently typed and as a result causes errors without defining a type for it.  #15057 is currently in PR and adds types for IconButton allowing us to remove these lines then.  As a result I can either wait on #15057 and remove these lines, or continue with this approach and clean up this todo in the future.

Reply:
also, should i change the extends for the Copy component as well to extend Carbon button rather than the native button

Reply:
@Tresau-IBM, sorry for the delay on this one. Now that #15057 has been merged, can we use `IconButton`?